ml-agents/mlagents/trainers/sac/policy.py

```diff
@@ -188,7 +188,12 @@ def evaluate(self, brain_info: BrainInfo) -> Dict[str, np.ndarray]:
 
     @timed
     def update(
-        self, mini_batch: Dict[str, Any], num_sequences: int, update_target: bool = True
+        # pylint: disable=arguments-differ
+        # TODO ervteng FIX ME
+        self,
```

Comment:
@ervteng Can you take a look at this? `update_target` is not in the base class signature. I only did a quick search, but it looked like we only ever call this with `update_target=True` but wasn't 100% sure if it was safe to remove

Reply:
Hmm, this is a subtlety of the SAC algorithm. There are two value networks, a target and a current one. The target is used to bootstrap the current one, and is updated periodically towards the weights of the current. 

Most implementations of SAC let you vary when this update happens. Initially, I had an additional hyperparameter that did just that, but removed it b/c realistically you can get away with just adjusting tau (and less hyperparameters are better for most users). 

I think it's safe to remove it for now. A researcher might want to vary this, but then again a researcher can edit the Python code. Perhaps we can add a comment -> I'll add it below.

Reply:
Line 213: `# Update target network. By default, target update happens at every policy update.`

Reply:
Sounds good, I'll apply that. If you want to keep the flexibility, you could so something like move all the current `update()` to `_update()`, and make `update()` call `_update()` with `update_target=True` (or some other logic)

Reply:
👍 I'll probably do something like this when we rehaul the Policy and Trainer APIs, but will hold off for this particular version of the code.